templates/webapi/test/overview.html.ep

```diff
@@ -147,6 +147,27 @@
                         </div>
                     </div>
                 </div>
+                <div class="row" id="filter-job-groups">
+                    <div class="col-5">
+                        <div class="form-group">
+                            <label for="group-glob"><strong>Include job groups</strong></label>
+                            <%= help_popover('Help for the <em>Include job group</em> filter' => '
+                            <p>Shows jobs from job groups matching these globs</p>') %>
+                            <input name="group_glob" type="text" id="group-glob"
+                            placeholder="globs, comma separated, e.g. *kernel*,*Tumbleweed*"
```

Comment:
The default globs are not very creative, there's probably better ones to show. 🤔